pemFioi/conceptDisplay-1.0.js

```diff
@@ -30,6 +39,38 @@ function conceptDisplay() {
       allLangDivs.show();
     }
   }
+
+  if (channel) {
+    var pythonCodes = $('.pythonCode');
+    pythonCodes.each(function (index, element) {
+      const jQueryElement = $(element);
+      if (!jQueryElement.parent().find('.pythonCode-execute').length) {
+        const previousCode = $(element).html();
+        jQueryElement
+          .removeClass('pythonCode')
+          .addClass('pythonCode-container')
+          .empty()
+          .append($("<div class='pythonCode'></div>").html(previousCode))
+          .append("<button class='pythonCode-execute'>Utiliser cet exemple</button>")
+      }
+    })
+
+    $('.pythonCode-execute').click(function () {
+      const code = $(this).parent().find('.pythonCode').text();
+
+      channel.call({
+        method: 'useCodeExample',
+        params: {
+          code,
+          language: 'python',
+        },
+        success: function () {
+        },
+        error: function () {
+        }
```

Comment:
If you don't care about the success/error result, you can use `channel.notify` instead of `channel.call`.